src/inmanta/module.py

```diff
@@ -39,6 +39,7 @@
 from enum import Enum
```

Comment:
Note, I got here, but I'm going to wait with the res until we are certain the ` SafeRequirement` is here to stay, because it makes the review very hard